domains/transfer/Shared/DataContext/Migrations/20240307132055_AddReceiverNameToTransferAgreements.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace DataContext.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddReceiverNameToTransferAgreements : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
```

Comment:
I would like a migration test for this

Reply:
I will give you two :)

https://github.com/Energinet-DataHub/energy-origin/pull/773/commits/6c6f2f5deb1965498481108106a94abb1d14295f